FilterList/FilterList/interfaces/interfaces.d.ts

```diff
@@ -15,9 +15,10 @@ export interface Expression {
   name: string;
   definitionExpression?: string;
   type?: string;
-  field?: "string" | "number" | "date";
+  field?: "string" | "number" | "date" | "coded-value" | "range";
```

Comment:
this type is getting pretty long maybe create a seperate type for it?
`type Field = "string" | "number" | "date" | "coded-value" | "range"`